core/native-bridge.ts

```diff
@@ -0,0 +1,566 @@
+import type {
+  CallData,
+  CapacitorInstance,
+  ErrorCallData,
+  MessageCallData,
+  PluginResult,
+  WindowCapacitor,
+} from './src/definitions-internal';
+
+// For removing exports for iOS/Android, keep let for reassignment
+// eslint-disable-line
+const dummy = {};
+
+const initBridge = (w: any): void => {
+  const getPlatformId = (win: WindowCapacitor): 'android' | 'ios' | 'web' => {
```

Comment:
since it's a .ts file now, we can use the getPlatformId from util instead of duplicating